nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -172,12 +172,6 @@ impl Executable {
             stmt.compile(&mut ctx);
         }
 
-        if ctx.exe.instructions.last() != Some(&Instruction::Return.as_u8()) {
```

Comment:
**suggestion:** Would it be simpler to keep injecting this but only based on a boolean parameter (or a boolean in `exe`) that is passed `true` for scripts and concise body arrow functions?

Reply:
Done.